docs/tutorials/setup-opentelemetry.md

```diff
@@ -24,16 +24,18 @@ yarn --cwd packages/backend add @opentelemetry/sdk-node \
 
 ## Configure
 
-In your `packages/backend/src` folder, create an `instrumentation.ts` file.
+In your `packages/backend` folder, create an `instrumentation.ts` file.
```

Comment:
I am not sure that this should be a .ts file, right? Sounds to me like this should be js

Reply:
Yeah, .js should be enough